src/genie/libs/parser/hp_comware/show_interface.py

```diff
@@ -0,0 +1,703 @@
+                                                                 'Link',
+                                                                 'Protocol',
+                                                                 'Main IP',
+                                                                 'Description'],
+                                                  label_fields=['Interface',
+                                                                'link',
+                                                                'protocol',
+                                                                'ip_address',
+                                                                'description'],
+                                                  index=[0])
+                if res.entries:
+                    for intf, intf_dict in res.entries.items():
+                        intf = Common.convert_intf_name(intf)
+                        del intf_dict['Interface']
+                        route_dict.setdefault('route', {}).update({intf: intf_dict})
+            if (bridge_out != ''):
+                res = parsergen.oper_fill_tabular(device_output=bridge_out,
+                                                  device_os='hp_comware',
+                                                  table_terminal_pattern=r"^\n",
+                                                  header_fields=['Interface',
+                                                                 'Link',
+                                                                 'Speed',
+                                                                 'Duplex',
+                                                                 'Type',
+                                                                 'PVID',
+                                                                 'Description'],
+                                                  label_fields=['Interface',
+                                                                'link',
+                                                                'speed',
+                                                                'duplex',
+                                                                'type',
+                                                                'pvid',
+                                                                'description'],
+                                                  index=[0])
+                if res.entries:
+                    for intf, intf_dict in res.entries.items():
+                        intf = Common.convert_intf_name(intf)
+                        del intf_dict['Interface']
+                        bridge_dict.setdefault('bridge', {}).update({intf: intf_dict})
+
+            if(route_dict != {}):
+                parsed_dict.update(route_dict)
+            if(bridge_dict != {}):
+                parsed_dict.update(bridge_dict)
+
+        return (parsed_dict)
```

Comment:
No need for brackets I believe;
`return parsed_dict`

Reply:
Fixed